README.md

```diff
@@ -55,6 +63,64 @@ validate('invalid_dl');
 // => null
 ```
 
+## validateForState
+
+```js
+import { validateForState } from 'driver-license-validator';
+
+validateForState('A1234567', 'CO');
```

Comment:
What's the difference between this new `validateForState` function and the current `validate` with the `states` option?

`validate('A1234567', { states: 'CO' });`

Reply:
It returns a boolean for a specific state.

I don't necessarily mind the `validate` function and using it as is. But, you'll have to check for null or the presence of the returned validation object (which in my mind doesn't really make, sense, you're asking "is this valid" and returning "yes, it's valid, here are the parameters in which it is valid for", but you'd have to know know what those are before asking?) It feels like a funky API in my mind. 

In my mind the flow of data here is backwards I feel like the way this package is written, you'd type in a dl number and then get a list of valid states. But, you're only gonna have one valid dl, so just select the state (get the requirementsByState) then validate by that specific state (validateForState) and return a boolean.

Does that all make sense?

Reply:
I also realize this is a tiny package and used in a very limited manner. I realize I pretty much rewrote it to suit my specific needs, so you're welcome to tell me to bugger off. :joy:

Reply:
Hmm, interesting point! It might be best if the current `validate` method was renamed (something like `getMatches`, `findMatches`, `getMatchingStates`, etc) and the new method was called `isValid`. I would like to keep the signatures (1st arg is value, 2nd arg os options object) for both functions the same though for consistency. 

Do you think that would make for a more straightforward API?